src/java.desktop/share/classes/javax/swing/text/html/AccessibleHTML.java

```diff
@@ -742,9 +740,9 @@ private ElementInfo getElementInfoAt(ElementInfo elementInfo, Point p) {
          * @see AccessibleStateSet
          */
         public boolean isFocusTraversable() {
-            Component comp = getTextComponent();
-            if (comp instanceof JTextComponent) {
-                if (((JTextComponent)comp).isEditable()) {
+            JTextComponent comp = getTextComponent();
+            if (comp != null) {
+                if (comp.isEditable()) {
```

Comment:
is it possible to merge two "if"? or even "return comp != null && comp.isEditable();"?